de.variantsync.core/src/main/java/de/variantsync/core/ast/AST.java

```diff
@@ -9,34 +9,34 @@
 import de.variantsync.core.interfaces.Grammar;
 
 /**
- * This class represents the Abstract Syntax Tree data structure.
+ * This class represents the Abstract Syntax Tree data structure. This also tests the LineGrammar indirectly.
```

Comment:
How and why does it test the LineGrammar? It does not appear here anywhere, right?